src/ica_api.c

```diff
@@ -3312,16 +4128,22 @@ int ica_aes_gcm_kma_verify_tag(const unsigned char* known_tag, unsigned int tag_
 		return EFAULT;
 
 	return 0;
+#endif /* NO_CPACF */
 }
 
 void ica_aes_gcm_kma_ctx_free(kma_ctx* ctx)
 {
+#ifdef NO_CPACF
```

Comment:
Do we really need this here?  The function is safe for calls with `NULL`.  So I think we could just use this function always in the same way without the `#ifdef NO_CPACF`.

Reply:
Good catch! It's the only function that can indeed be used without a change in libica-cex. 
Done.